docs/changelog/91788.yaml

```
@@ -0,0 +1,5 @@
+pr: 91788
+summary: Skip remote clusters when performing privileges check
```

I think this needs to be rephrased to avoid ringing alarm bells that we're bypassing security. We're _not_ bypassing security, just having the security checks applied at a later time when diagnosis of any problems is harder.

```suggestion
summary: Skip remote clusters when performing up front privileges validation
```